cmd/plume/prerelease.go

```diff
@@ -692,6 +701,25 @@ func awsPreRelease(ctx context.Context, system string, client *http.Client, src
 	imageDescription := fmt.Sprintf("%v %v %v", spec.AWS.BaseDescription, specChannel, specVersion)
 
 	if system == "fedora" {
+		imageFileNameTmpl := spec.AWS.Image
+		t := template.Must(template.New("filename").Parse(imageFileNameTmpl))
+		if specChannel == "updates" || specChannel == "twoweek" {
+			builder := &strings.Builder{}
+			if err := t.Execute(builder, imageMetadata); err != nil {
+				panic(err)
+			}
+			s := builder.String()
+			fmt.Println(s)
+
+		} else if specChannel == "version" {
+			fmt.Println("In version")
```

Comment:
Debug code

Reply:
Again this would resolve when the commits would be sorted.

Reply:
Debug code shouldn't be included in any commit.